test/react-svg-path-precision.js

```diff
@@ -13,43 +13,43 @@ const rule = rules["react-svg-path-precision"];
 ruleTester.run("react-svg-path-precision", rule, {
     valid: [
         {
-            code: '<path d="M1.23,0.45L-0.98,42Z"/>',
```

Comment:
Hmm - it looks like we lost some of the leading 0s - is that ok? seems a bit scary.

Reply:
That's intentional.  I want to check that the code change handle that case properly.